specification/cognitiveservices/data-plane/TranslatorText/readme.md

```diff
@@ -0,0 +1,130 @@
+# Cognitive Services Translator Text SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Translator Text SDK.
+
+The current release is `release_3_0`.
+
+``` yaml
+
+tag: release_3_0
+add-credentials: true
+openapi-type: data-plane
+```
+
+# Releases
+
+### Release 3.0
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: stable/v3.0/TranslatorText.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-nodejs
+```
+
+## CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+``` yaml $(csharp)
+csharp:
+  sync-methods: None
+  license-header: MICROSOFT_MIT_NO_VERSION
+  azure-arm: false
+  namespace: Microsoft.Azure.CognitiveServices.TranslatorText
+  output-folder: $(csharp-sdks-folder)/CognitiveServices/dataPlane/TranslatorText/TranslatorText/Generated
+  clear-output-folder: true
+```
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  add-credentials: true
+  payload-flattening-threshold: 2
+  namespace: azure.cognitiveservices.translatortext
+  package-name: azure-cognitiveservices-translatortext
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-cognitiveservices-translatortext/azure/cognitiveservices/translatortext
+```
+``` yaml $(python) && $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/azure-cognitiveservices-translatortext
+```
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: translatortext
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: release_3_0 and go
```

Comment:
The tag name is wrong it should be `release_3_0`

Reply:
I removed ` and go` see if it works